internal/cmd/gocdk/internal/static/_assets/provision/blob/fileblob.tf

```diff
@@ -1,7 +1,7 @@
 # TODO(rvangent): Add comments explaining.
 
 locals {
-  blob_bucket_path    = "${path.module}/fileblob_scratchdir"
+  blob_bucket_path    = "${path.cwd}/fileblob_scratchdir"
```

Comment:
This is a behavior change and I don't believe it is correct: `${path.module}` stores the directory inside the biome whereas `${path.cwd}` stores it wherever the `terraform apply` is run from.

Reply:
Hmm, I'm not sure what to do then. The output of `${path.module}` changed in this Terraform version; it is no longer giving an absolute path, but a relative one, resulting in:

```
  "BLOB_BUCKET_URL" = "file://./fileblob_scratchdir"
```

which does not work. I haven't been able to find a Terraform function that makes a path absolute.

Reply:
Given that we run Terraform, `cwd` seems like it works...

Reply:
That's.... really unfortunate. That feels like a Terraform bug.

Reply:
Maybe. I don't think they really want to support local files since they have weird behavior for multiple users sharing `.tfstate`.